rm: cannot remove 'CPAR2_602950-T.tmp.out': No such file or directory
Traceback (most recent call last):
  File "get_combined_score.py", line 1023, in <module>
    main()
  File "get_combined_score.py", line 699, in main
    rnn = open("%s.RNN.tab_tmp" %molname)
IOError: [Errno 2] No such file or directory: 'CPAR2_602950-T.RNN.tab_tmp'

real	0m17.520s
user	0m10.173s
sys	0m1.906s
mv: cannot strm: cannot remove 'CPAR2_602950-T.tmp.out': No such file or directory
rm: cannot remove '/gpfs/projects/bsc40/umv: cannot stat '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/seq/CPAR2_602950-T.outfile': No such file or directory
Traceback (most recent call last):
  File "/gpfs/projects/bsc40/uchorostecki/software/MultiFolds/scripts/sliding_windows_nextPARSE_score.py", line 58, in <module>
    main()
  File "/gpfs/projects/bsc40/uchorostecki/software/MultiFolds/scripts/sliding_windows_nextPARSE_score.py", line 54, in main
    process_tab(args.infile,int(args.window),int(args.step))
  File "/gpfs/projects/bsc40/uchorostecki/software/MultiFolds/scripts/sliding_windows_nextPARSE_score.py", line 29, in process_tab
    df = read_csv(tab_file,header=None,delimiter='\t',names = ['name', 'data'])
  File "/apps/PYTHON/2.7.14/INTEL/lib/python2.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/apps/PYTHON/2.7.14/INTEL/lib/python2.7/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/apps/PYTHON/2.7.14/INTEL/lib/python2.7/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/apps/PYTHON/2.7.14/INTEL/lib/python2.7/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/apps/PYTHON/2.7.14/INTEL/lib/python2.7/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
IOError: [Errno 2] File CPAR2_602950-T.RNN_NP_ONLY.csv does not exist: 'CPAR2_602950-T.RNN_NP_ONLY.csv'

real	0m0.054s
user	0m0.015s
sys	0m0.031s
./multifolds_cluster.sh: line 120: CPAR2_602950-T.fa: No such file or directory
split: cannot open 'CPAR2_602950-T.SHAPE' for reading: No such file or directory
ls: cannot access '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/res/splited_*': No such file or directory
mv: cannot stat '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/res/*_*.SHAPE': No such file or directory
mv: cannot stat '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/res/*_*.fa': No such file or directory
ls: cannot access '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/final/*SHAPE': No such file or directory
ls: cannot access '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/final/*.ct': No such file or directory
PAR2_602950-T_23_180_60/res/*_*.SHAPE': No such file or directory
mv: cannot stat '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/res/*_*.fa': No such file or directory
ls: cannot access '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/final/*SHAPE': No such file or directory
ls: cannot access '/gpfs/projects/bsc40/uchorostecki/projects/MULTI-FOLDS/multiple_conformations/23_degrees/CPAR2_602950-T_23_180_60/final/*.ct': No such file or directory
